cmd/nerdctl/container_run_systemd_linux_test.go

```diff
@@ -46,7 +46,16 @@ func TestRunWithSystemdTrueEnabled(t *testing.T) {
 
 	base.Cmd("inspect", "--format", "{{json .Config.Labels}}", containerName).AssertOutContains("SIGRTMIN+3")
 
-	base.Cmd("exec", containerName, "systemctl", "list-jobs").AssertOutContains("jobs listed.")
+	base.Cmd("exec", containerName, "sh", "-c", "--", `tries=0
+until [ -e /run/systemd/io.system.ManagedOOM ]; do
```

Comment:
Would it be possible to use `systemctl is-system-running`?

Reply:
Of course. I am just not that familiar with systemd TBH, so, whatever is best in their world.

Will use your suggestion ^.

Reply:
Modified.